eng/platform-matrix.yml

```diff
@@ -1,149 +1,138 @@
 parameters:
   jobTemplate: ''
   buildConfig: ''
-  platforms:
-  - Linux_arm
-  - Linux_arm64
-  - Linux_musl_x64
-  - Linux_musl_arm64
-  - Linux_rhel6_x64
-  - Linux_x64
-  - OSX_x64
-  - Windows_NT_x64
-  - Windows_NT_x86
-  - Windows_NT_arm
-  - Windows_NT_arm64
+  platforms: []
+  platformGroup: ''
```

Comment:
It would be useful to have a comment documenting `platformGroup`, e.g., what values are allowed, and what the intention is for each possible value.